metadata-ingestion/source_docs/ldap.md

```diff
@@ -51,11 +69,37 @@ Note that a `.` is used to denote nested fields in the YAML recipe.
+| `attrs_mapping.givenName`      |          | `givenName`         | Alternate attrs key representing same information as givenName in the organization. |
+| `attrs_mapping.sn`             |          | `sn`                | Alternate attrs key representing same information as sn in the organization. |
+| `attrs_mapping.cn`             |          | `cn`                | Alternate attrs key representing same information as cn in the organization. |
+| `attrs_mapping.mail`           |          | `mail`              | Alternate attrs key representing same information as mail in the organization. |
+| `attrs_mapping.displayName`    |          | `displayName`       | Alternate attrs key representing same information as displayName in the organization. |
+| `attrs_mapping.departmentNumber`|          | `departmentNumber` | Alternate attrs key representing same information as departmentNumber in the organization. |
+| `attrs_mapping.title`          |          | `title`             | Alternate attrs key representing same information as title in the organization. |
+| `attrs_mapping.owner`          |          | `owner`             | Alternate attrs key representing same information as owner in the organization. |
+| `attrs_mapping.managedBy`      |          | `managedBy`         | Alternate attrs key representing same information as managedBy in the organization. |
+| `attrs_mapping.uniqueMember`   |          | `uniqueMember`      | Alternate attrs key representing same information as uniqueMember in the organization. |
+| `attrs_mapping.member`         |          | `member`            | Alternate attrs key representing same information as member in the organization. |
 
 The `drop_missing_first_last_name` should be set to true if you've got many "headless" user LDAP accounts
 for devices or services should be excluded when they do not contain a first and last name. This will only
 impact the ingestion of LDAP users, while LDAP groups will be unaffected by this config option.
 
+### Configurable LDAP
+
+Every organization may implement LDAP slightly differently based on their needs. The makes a standard LDAP recipe ineffecvtive due to missing data during LDAP ingestion. For instance, LDAP recipe assumes department information for a CorpUser would be present in the `departmentNumber` attribute. If an organization chose not to implement that attribute or rather capture similar imformation in the `department` attribute, that information can be missed during LDAP ingestion (even though the information may be present in LDAP in a slightly different form). LDAP source provides flexibility to provide optional mapping for such variations to be reperesented under attrs_mapping. So if an organization represented `departmentNumber` as `department` and `mail` as `email`, the recipe can be adapted to customiza that mapping based on need. An example is show below. If `attrs_mapping` section is not provided, the default mapping will apply.
```

Comment:
minor typo: 

ineffecvtive -> ineffective

Reply:
typo:

customiza -> customize

Reply:
Thanks for catching these!